docs/content/preview/releases/yba-releases/v2.20.md

```diff
@@ -23,6 +23,29 @@ What follows are the release notes for all releases in the **YugabyteDB Anywhere
 
 For an RSS feed of all release series, point your feed reader to the [RSS feed for releases](../../index.xml).
 
+## v2.20.0.1 - December 5, 2023 {#2.20.0.1}
+
+**Build:** `2.20.0.1-b1`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.20.0.1/yugabytedb-2.20.0.1-b1-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.20.0.1/yugabytedb-anywhere-2.20.0.1-b1-third-party-licenses.html)
+
+### Docker
```

Comment:
If you look at 2.19 page, we replaced this section with link to YBA Install instructions